examples/flag_persistence_generators.ipynb

```diff
@@ -0,0 +1,168 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a40d1d2e-c23e-4a89-9a0c-694ce37fb8f0",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Flag persistence generators\n",
+    "\n",
+    "In this notebook, we show how to retrieve the vertices and edges responsible for the creation/destruction of persistent topological features in a Vietoris–Rips filtration. We use the same setup and kind of dataset as in the [basic tutorial](https://github.com/giotto-ai/giotto-ph/blob/main/examples/basic_tutorial.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c821577f-4928-4403-90c3-d4b6ef4c5440",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install missing dependencies\n",
+    "import sys\n",
+    "!{sys.executable} -m pip install giotto-tda"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7e4bf17b-f6b4-444c-a344-38ff096c7f4a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Giotto-ph\n",
+    "from gph.python import ripser_parallel\n",
```

Comment:
The import here was not updated correctly, I wonder if something went wrong in resolving conflicts @MonkeyBreaker ?

Reply:
I fixed it, and I suspect nothing went wrong. Merging!

Reply:
:angry: I failed the merge, sorry for that